api/package.json

```diff
@@ -66,17 +63,17 @@
     "node": "^10.14"
   },
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-config-prettier": "^2.10.0",
-    "eslint-plugin-import": "^2.14.0",
-    "jest": "^23.6.0",
-    "nodemon": "^1.18.6",
+    "eslint": "^5.14.1",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-import": "^2.16.0",
```

Comment:
Flagging because `eslint` was one of the things you mentioned would be done in another issue.

Reply:
Good catch!  I should have been clearer - updating eslint on the frontend.  On the backend there were maybe a half dozen things it wanted fixed, so I did it.  On the frontend, though...  woof.